cfgov/unprocessed/apps/owning-a-home/css/explore-rates.less

```diff
@@ -515,6 +569,28 @@
         margin-top: @grid_gutter-width;
     }
 
+    .note-ctr {
+        border-top: 1px solid @gray-10;
+    }
+
+    .note {
+        .grid_column( 8 );
+        margin: 0;
+        padding: 0 0 0 25px;
+
+        .alert:before {
+            color: @gold;
+        }
+
+        p {
+            width: 100%
```

Comment:
MINOR: need a semicolon. I guess less doesn't need them hm.

Reply:
fixed in https://github.com/cfpb/cfgov-refresh/pull/4047/commits/8bbb6446b79b651eaf9156dbecdeef7576d076be;